build/root/Makefile

```diff
@@ -203,7 +203,7 @@ define TEST_IT_HELP_INFO
 # Example:
 #   make test-integration
 #   make test-integration WHAT=./test/integration/kubelet GOFLAGS="-v -coverpkg=./pkg/kubelet/..." KUBE_COVER="y"
-#   make test-integration WHAT=./test/integration/pods GOFLAGS="-v" KUBE_TEST_ARGS='-run ^TestPodUpdateActiveDeadlineSeconds$$'
```

Comment:
I suspect that at some point, the double `$$` was necessary because these args where passed through shell unescaping twice. Now that's indeed not necessary anymore.

Thanks for taking the time to fix this. It'll help the next person who stumbles over this.

/lgtm
/approve